skorch/callbacks/lr_scheduler.py

```diff
@@ -116,7 +116,11 @@ def kwargs(self):
 
     def on_train_begin(self, net, **kwargs):
         if net.history:
-            self.batch_idx_ = len(net.history[:, 'batches']) - 1
+            try:
+                self.batch_idx_ = sum(net.history[:, 'train_batch_count'])
+            except KeyError:
+                self.batch_idx_ = sum(len(b)
```

Comment:
Very minor nitpick: I don't like the indentation here, this should fit in one line (88 chars is okay).

Reply:
I have gotten too use to sklearn's 80 characters limit. :)

This PR has been updated.